cli-output/src/cli_output.rs

```diff
@@ -107,6 +108,18 @@ pub struct CliAccount {
     pub use_lamports_unit: bool,
 }
 
+impl CliAccount {
+    pub fn new<T: ReadableAccount>(address: &Pubkey, account: &T, use_lamports_unit: bool) -> Self {
```

Comment:
add a config to expose the other `UiAccount` fields or nah?

Reply:
I'm surprised your use case doesn't call for it yet, but if you don't need it, fine to kick the can.
Easy enough to add later as `new_with_config()` or something like that.

Reply:
ok i lied.  totally ended up needing it :sweat_smile: 

https://github.com/solana-labs/solana/pull/28646